raiden/tests/utils/mock_client.py

```diff
@@ -268,7 +272,7 @@ def manager_addresses(self):
             for manager in self.token_manager.values()
         ]
 
-    def tokenadded_filter(self):
+    def tokenadded_filter(self, from_block='', to_block=''):
```

Comment:
Good point. The only reason I kept it like that is because these are the default values we were forwarding to the filter before.